### PR TITLE
Add Contentful service (base) 1/2

### PR DESCRIPTION
**Pull request by thibmaek**

### CODE REVIEW

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-86-contentful-integration

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kevinmeyvaert** [comment]
I think you might have a linting error somewhere resulting the build to fail :)

File: README.md [hunk 1]
```diff
@@ -26,8 +26,12 @@ When you completed an issue, you can open a pull request to develop.
  * Install dependencies `npm install`
  * Run project `npm start`
 
+## Setup config (env)
 
-## Setup & Hook a firebase database
+* Rename `config/env.sample.json` to `config/env.json` and fill in your keys & constants
+* To avoid going over the rate limit for Contentful's free plan you can run a script which will export your entries to a json file which will be used if you set the `workingLocally` property to true in `config/env.json`. To generate a dump you can run `node scripts/contentful-export.js` and it will output your entries to a json file in the scripts folder. (⚠️ STILL EXPERIMENTAL)
```
Comment (easybird) [inline README.md]: so the app cannot be used (i see you removed the fb app id?) without doing this step?
The app should be usable without adding own config.

File: types/contentful.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      [TLocale]: TDateLike,
+    },
+    description: {
+      [TLocale]: string,
+    },
+    location: {
+      [TLocale]: string,
+    },
+    speakerImage: {
+      [TLocale]: TContentfulAsset,
```
Comment (easybird) [inline types/contentful.js]: this is the "modelled" data?  The data that will be used inside the app? I'd like to keep all contentful references outside of the "modelled data".

File: utils/contentfulService.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import type { TImageObj } from '../types/primitives';
+import contentfulMock from '../scripts/_contentful-export.json';
+// import getDeviceLocale from './getDeviceLocale';
+
+export const normalizeContentfulEntries = (entries: TContentfulEntries) =>
+  entries.map((entry) => {
+    if (!entry.hasOwnProperty('fields')) return entry;
+
+    // TODO: get locale and from Contentful only return the keys with this locale
+    // const locale = await getDeviceLocale();
```
Comment (easybird) [inline utils/contentfulService.js]: what should be the reason not to load in all locales, apart from the deviceLocale? (two locales).
Imo we will ship the data with the app at first, and do OTA updates if needed. Apart from the user his device locale.

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-86-contentful-integration

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-86-contentful-integration

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-86-contentful-integration

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: screens/__tests__/HomeContainer.test.js [hunk 4]
```diff
@@ -7,18 +7,25 @@ jest.mock('../../utils/firebaseService');
...
+        />
+      );
+      const tree = renderer.create(wrapWithStateManagement(comp)).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+  });
+
+  describe('with tracks', () => {
+    beforeAll(() => {});
+    it('renders correctly.', () => {});
```
Comment (easybird) [inline screens/__tests__/HomeContainer.test.js]: das hier iets raars @thibmaek 👎

File: types/contentful.js (hunk 2 above)
Comment (easybird) [reply types/contentful.js]: @thibmaek pls?

Comment (thibmaek) [reply types/contentful.js]: Ok

**Pull request merged**